x-pack/plugins/fleet/server/collectors/agent_logs_top_errors.ts

```diff
@@ -29,7 +30,8 @@ export async function getAgentLogsTopErrors(
     const queryTopMessages = (index: string) =>
```

Comment:
I am wondering if should do the same here as we did for the agent metrics to only query hot shard, to avoid any performance issue?

Reply:
Good catch, I'll add the filter.